skiko/src/commonMain/kotlin/org/jetbrains/skiko/Events.kt

```diff
@@ -8,14 +8,36 @@ object SkikoMouseButtons {
     const val MIDDLE = 1 shl 2
 }
 
+enum class SkikoGestureEventKind {
+    PRESS, TAP, PAN, PINCH, ROTATION, LONGPRESS, SWIPE, UNKNOWN
+}
+enum class SkikoGestureEventDirection {
+    UP, DOWN, LEFT, RIGHT, UNKNOWN
+}
+enum class SkikoGestureEventState {
+    PRESS, STARTED, CHANGED, ENDED, UNKNOWN
+}
+expect class SkikoGesturePlatformEvent
+data class SkikoGestureEvent(
+    val x: Double, val y: Double,
+    val velocity: Double = 0.0,
+    val direction: SkikoGestureEventDirection = SkikoGestureEventDirection.UNKNOWN,
+    val interval: Long = 0,
```

Comment:
Do we need this property?

Reply:
`interval`? `LONGPRESS` gesture has it, but if compose uses raw touch events, we probably don't need it.

Reply:
It is forgotten [here](https://github.com/JetBrains/skiko/blob/78584673756f3cf05f249d1855a3c448ee43d368/skiko/src/iosMain/kotlin/org/jetbrains/skiko/SkiaLayer.ios.kt#L81)